set.mm

```diff
@@ -515739,6 +515739,119 @@ The sumset operation can be used for both group (additive) operations and
+        UUEUWGUWCUWEUVBDWQUFZUDZCUFZUWHJUWCUUPUWEUUKTUVBUUKTZUWMUWHUPAUUPUWBMWR
+        ZUWCUUKDUWDIUVJUWDVRZUWCUUPUUQUWOUURVHZAUUMUWBSWRZWSAUUBUUKUVBUVTWTZUWK
+        UWGDEUWECUVBUUKUVJUWKVRZUWGVRZXAXBUWCUWLUUKTZUWMJUPZUWCUVGUWLCXDJVMUGZT
+        ZUXBUXCWNZAUVGUWBUVMWRUWCHUXDUWLAHUXDVPUWBAHGUXDQNXCWRUWCUUQHUUKVPZIUVB
+        UUAXEZUWLHTZUWQAUXGUWBAUVNUVOUXGRUVQUUKHDUVJXFWHWRAUUMUWBUXHSUUMUWBUXHU
+        VBUKTUUMUWBUXHVQUBXOUVBIUUAUKUUKXGXHXIXJUUQUXGWNZUXHWNUUMUWNUXIUXJUXHUU
+        MUWNUXIXPIUVBUWKUUAHDUWDVAUUKUVJUWPUWTUVRXKXIXLXMWJUVGUXEUXFUUKJUWLCXNX
+        IWMXQXRXSUWCUWIUUEUUEEWPUFZUFZUVCUWGUDZUWGUDZUVCUWCUWHUXMUUEUWGUWCUWFUX
+        LUVCUWGUWCUUPUUMUWFUXLUPUWOUWRUUKDECUWDUXKIUVJUWPUXKVRZXTWMYAXSUWCEVATZ
+        UUEUVHTZUVCUVHTUXNUVCUPUWCUUPUXPUWODECYBVHZUWCUUKUVHICUWCUUPUVIUWOUVLVH
+        ZUWRYCZUWCUUKUVHUVBCUXSUWSYCUVHUWGEUXKUUEUVCUVKUXAUXOYDXBYEUWCUXPUXQUWJ
+        UUEUPUXRUXTUVHUWGEUUEJUVKUXALYFWMYGYHWRXRYIAUWAWNZUVDUUEUUTUPZUBIUUBAIU
+        UBTZUWAAUVPUUMUYCUVSSIUUAUUKYJWMWRUVBIUPUVDUYBVQUYAUVBIUUTCYKUQUYAUUTUU
+        EAUWAWOYHYLYOUUTUUEUAXOYMYPYNYQUOUUEICYRYSYTYE $.
+    $}
+
+    ${
+      $d F q $.  $d G q x $.  $d K q $.  $d K x $.  $d N q $.  $d N x $.
+      $d Q q $.  $d Y q $.  $d Y x $.  $d ph q $.  $d ph x $.
+      ghmqusnsglem2.y $e |- ( ph -> Y e. ( Base ` Q ) ) $.
+      $( Lemma for ~ rhmqusnsg .  (Contributed by Thierry Arnoux,
+         13-May-2025.) $)
+      ghmqusnsglem2 $p |- ( ph -> E. x e. Y ( J ` Y ) = ( F ` x ) ) $=
+        ( cv cqg co cec wceq cbs cfv wrex wcel cqs cvv cgrp cqus a1i eqidd cghm
+        ovexd ghmgrp1 syl qusbas eleqtrrd elqsg biimpa syl2anc wa csubg nsgsubg
+        wer cnsg eqid eqger ad2antrr simplr ecref simpr wss ghmqusnsglem1 eqtrd
+        fveq2d jca anasss ex reximdv2 mpd ) AJBUAZEIUBUCZUDZUEZBEUFUGZUHZJGUGZW
+        EDUGZUEZBJUHAJCUFUGZUIZJWIWFUJZUIZWJTAJWNWPTAWFECWIUKULCEWFUMUCUEAPUNAW
+        IUOAEIUBUQADEFUPUCUIZEULUINEFDURUSUTVAWOWQWJBWIJWFWNVBVCVDAWHWMBWIJAWEW
+        IUIZWHVEWEJUIZWMVEZAWSWHXAAWSVEZWHVEZWTWMXCWEWGJXCWIWFVHZWSWEWGUIAXDWSW
+        HAIEVFUGUIZXDAIEVIUGUIZXESIEVGUSWFEWIIWIVJWFVJVKUSVLAWSWHVMZWEWFWIVNVDX
+        BWHVOZVAXCWKWGGUGWLXCJWGGXHVSXCCDEFGHIWEKLMAWRWSWHNVLOPQAIHVPWSWHRVLAXF
+        WSWHSVLXGVQVRVTWAWBWCWD $.
+    $}
+
+    $d .0. r x y $.  $d F q r x y $.  $d F r x y z $.  $d G q r x y $.
+    $d G z $.  $d H q r s x y $.  $d H s z $.  $d J q r s x y $.  $d J s z $.
+    $d K q r x y $.  $d K z $.  $d N q x y $.  $d Q q r s x y $.  $d Q s z $.
+    $d ph q r s x y $.
+    $( The mapping ` H ` induced by a surjective group homomorphism ` F ` from
+       the quotient group ` Q ` over a normal subgroup ` N ` of ` F ` 's kernel
+       ` K ` is a group isomorphism.  In this case, one says that ` F ` factors
+       through ` Q ` , which is also called the factor group.  (Contributed by
+       Thierry Arnoux, 13-May-2025.) $)
+    ghmqusnsglem3 $p |- ( ph -> J e. ( Q GrpHom H ) ) $=
```

Comment:
This is ~ ghmqusnsg, an important theorem by itself, not only a lemma (for ~rhmqusnsg).

Reply:
Yes: I've renamed the theorem in https://github.com/metamath/set.mm/pull/4824/commits/d1d759a69912464cc720e307ddb6e3cd0e68d536